Source/TeamMate/ViewModels/PullRequestPageViewModel.cs

```diff
@@ -63,7 +63,7 @@ private ListViewModel CreateListViewModel(ICollectionView collectionView)
             model.Filters.Add(new ListViewFilter("Pending", (o) => ((PullRequestRowViewModel)o).IsPending));
             model.Filters.Add(new ListViewFilter("Waiting", (o) => ((PullRequestRowViewModel)o).IsWaiting));
             model.Filters.Add(new ListViewFilter("Signed Off", (o) => ((PullRequestRowViewModel)o).IsSignedOff));
-            model.Filters.Add(new ListViewFilter("Not Signed Off By Me", (o) => !((PullRequestRowViewModel)o).IsSignedOffByMe));
+            model.Filters.Add(new ListViewFilter("Not Signed Off / Declined By Me", (o) => !((PullRequestRowViewModel)o).IsSignedOffOrDeclinedByMe));
```

Comment:
Why do you want to add the declined ones here too?
Typically when I decline the review, I assume the author is going to abandon it as we will not accept it?

Reply:
This will -ignore- declined ones. My goal for this one is that it should be the "actionable by me" list. It answers the "have I reviewed everything I should review for this query" question.